go.mod

```diff
@@ -85,7 +85,7 @@ require (
 	go.opentelemetry.io/otel/trace v0.20.0
 	go.opentelemetry.io/proto/otlp v0.7.0
 	go.uber.org/zap v1.19.0
-	golang.org/x/crypto v0.0.0-20220214200702-86341886e292
+	golang.org/x/crypto v0.0.0-20211202192323-5770296d904e
```

Comment:
oh I misunderstood it. The latest master's go.mod is normal, we should update 1.21, 1.22, 1.23 go.mod golang.org/x/crypto to v0.0.0-20211202192323-5770296d904e,is that right?

Reply:
1.22 and 1.23 are not impacted by the CVE since they don't link to the ssh package. 1.21 is, it looks like. I don't have a sense for the size/complexity of the backport, but I wouldn't expect to take a disruptive backport to releases that didn't need it.

Reply:
I see, this maybe only for 1.21.  So I should submit another PR for 1.21？this PR was originally intended to be merged into 1.24.

Reply:
> I see, this maybe only for 1.21. So I should submit another PR for 1.21？this PR was originally intended to be merged into 1.24.

yeah

Reply:
Thanks, I'll working on this.